src/main/java/com/wanderersoftherift/wotr/init/WotrBlocks.java

```diff
@@ -132,6 +132,114 @@ public class WotrBlocks {
                     .sound(SoundType.WOOD)
                     .noOcclusion()));
 
+    public static final DeferredBlock<Block> NOGRAVGRAVEL = registerBlock("nograv_gravel", () -> new Block(
```

Comment:
Create a static void method called registerNoGrav, which takes the idString and the SoundType as parameters and then calls registerBlock like here.